packages/gafl-webapp-service/src/pages/summary/licence-summary/route.js

```diff
@@ -45,6 +134,33 @@ export const checkNavigation = permission => {
   }
 }
 
+const getLicenceSummaryRows = (request, permission) => {
+  const rows = []
+  const { licensee } = permission
+
+  const rg = new RowGenerator(request, permission)
+  rows.push(rg.generateStandardRow('licence_summary_name', `${licensee.firstName} ${licensee.lastName}`, NAME.uri, 'change-name'))
+  if (!permission.isRenewal) {
+    const birthDateStr = moment(permission.licensee.birthDate).locale(request.locale).format('Do MMMM YYYY')
+    rows.push(rg.generateStandardRow('licence_summary_dob', birthDateStr, DATE_OF_BIRTH.uri, 'change-birth-date'))
+  }
+  rows.push(
+    rg.generateStandardRow(
+      'licence_summary_type',
+      licenceTypeDisplay(permission, request.i18n.getCatalog()),
+      LICENCE_TYPE.uri,
```

Comment:
should we have language code added on here? same for when calling other EXAMPLE.uri

Reply:
No, I add the language code on line 33, saves repeating it through the code then